### ISSUE TITLE
Copy of [CDP] Improve error messaging for a better screen reader user experience

### ISSUE THREAD

**Issue by josephrlee** [department-of-veterans-affairs/va.gov-team#88878]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

Issue Title: 508 Defect 4, Screen Readers, Cognition, error message can be improved for better user experience.
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

## User Story or Problem Statement:
As a screen reader user, it is easier to navigate error messages when they contain an action item I.E. "return to my VA debt," and when focus is placed on said action item when the error is triggered. 

## Details
In the event that there are no debt letters associated with a user account an error message appears on the screen. The screen reader announces the message in full, and keyboard focus is placed on the h1. The user can return to the previous page by navigating up to the breadcrumb.
User experience can be improved by including a "Return" link within the error message and moving focus to it upon the triggering of the message. 

## Tasks
- [x] Come up with a list of pages that need links back to the portal 
- [x] Define which links should go where
- [x] Update CDP figma file to reflect new links
- [x] Create annotations for dev/handoff

## Acceptance Criteria
> Generated by [Zenhub AI](https://www.zenhub.com/ai)

- [x] **Scenario 1: Improve error messaging for screen reader users**
  - Given there is an error message displayed on the screen
  - When the error message is triggered
  - Then the error message should include a "Return" link
  - And the focus should be moved to the "Return" link
